src/containerapp/azext_containerapp/tests/latest/test_containerapp_scenario.py

```diff
@@ -77,44 +78,17 @@ def test_containerapp_e2e(self, resource_group):
         ])
 
 
-    # TODO rename
-    @AllowLargeResponse(8192)
-    @ResourceGroupPreparer(location="eastus2")
```

Comment:
why remove the test

Reply:
There is another test with same name test_container_acr, the test case is duplicate.